kDrive/UI/View/Menu/MainTabBar.swift

```diff
@@ -16,6 +16,7 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import InfomaniakDI
```

Comment:
Will be used for the navigation code as soon as available

Reply:
Why do wee need to add an import that is useless for the moment?